### PR TITLE
Adding zappr config file

### LINKED ISSUE #235: Configure Zappr
We should enable Zappr, right?

### PR DESCRIPTION
**Pull request by Joneser**
Adding the zappr config file to resolve #235

### CODE REVIEW

File: .zappr.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+approvals:
+  minimum: 2
+  ignore: pr_opener
+  pattern: "^(:\\+1:|👍)$"
```
Comment (aryszka) [inline .zappr.yaml]: can we also use shipit and lgtm?

File: .zappr.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+approvals:
+  minimum: 2
```
Comment (aryszka) [inline .zappr.yaml]: i would bet 1 would be enough for an official approval

Comment (aryszka) [inline .zappr.yaml]: would add this:

```
  from:
    collaborators: true
```

or is this the default?

(https://zappr.readthedocs.io/en/latest/setup/)

Comment (Joneser) [inline .zappr.yaml]: It isn't there by default, i'll add along with your other suggestions

**Comment by Joneser** [comment]
@aryszka Config updated :)

**Comment by aryszka** [comment]
lgtm

**Pull request merged**